mantidproject/mantid
# Curvefitting code reorganistation

Moved the code aspects of CurveFitting into directories and namespaces to reduce overcrowding and add some order.

closes #13347
## Release Notes

Nothing to add
## To test
1. Check unit tests and system tests  pass
2. check basic usage within the GUI still works
